README.md

```diff
@@ -118,10 +118,10 @@ db.put('example', 42, function (err) {
 
 ### `const db = require('encoding-down')(db[, options])`
 
-* `db` must be an [`abstract-leveldown`] compliant store
-* `options` are passed to [`level-codec`]:
-  - `keyEncoding`: encoding to use for keys
-  - `valueEncoding`: encoding to use for values
+-   `db` must be an [`abstract-leveldown`] compliant store
+-   `options` are passed to [`level-codec`]&#x3A;
```

Comment:
Huh, remark encodes a colon?

Reply:
That was odd indeed

Reply:
I can remove the colon and re-render

Reply:
Nah, the colon makes sense here. We can live with an encoded thing in `encoding-down` :)